video/out/cocoa_cb_common.swift

```diff
@@ -122,7 +122,41 @@ class CocoaCB: Common {
 
         libmpv.setRenderICCProfile(colorSpace)
         if #available(macOS 10.11, *) {
-            layer?.colorspace = colorSpace.cgColorSpace
+
+            // Experimental #HDR on #macOS        
+            var name = "" as CFString
+
+            // HDR is supported fully only on macOS 10.15 and higher
+#if HAVE_MACOS_10_15_FEATURES
```

Comment:
for HAVE_MACOS_10_15_FEATURES to work you need to add a new configure flag for it to exists. otherwise this won't work. see https://github.com/mpv-player/mpv/blob/master/wscript#L816-L831